testapps/testapp/build.gradle

```diff
@@ -83,6 +87,9 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
+    buildFeatures {
```

Comment:
What does this do? and why is it needed?

Reply:
> What does this do
https://developer.android.com/topic/libraries/view-binding

> why is it needed?
It's a much improved process of managing view (interactions, etc.) that has been rolled out by Android for quite some time. New and improved way of doing things.